### ISSUE TITLE
Government

### ISSUE THREAD

**Issue by yellowgopher** [openpolitics/manifesto#312]
The more I look at the current structure of government and the policies that have been written in this manifesto to attempt to modify it, the more I pull my hair out!

The problem is we are falling into the same trap governments and political parties of all types have for, well ever really - at least since the formation of the modern UK parliamentary system.  We are tinkering with what we have, sometimes in a big way, sometimes less so.  But it feels patched together, somewhat contrived and just not satisfactory. (I should add this is no reflection on the authors, I know a lot of thought and effort has gone in, around deeply held beliefs.)

In many ways this"issue" goes hand in hand with the blank sheet constitution one... and may well be one and the same thing.

I think we need to be radical here (we can afford to be) and I think I may have an approach but it is a gamble.

We need three options:
- the status quo
- a modified Westminster system
- a new approach or approaches

In simple terms, the status quo would mean no change, a modified Westminster system would effectively be the same but with small modifications like reform of the House of Lords and the new approach/approaches; well I have some ideas but they would probably have to include the blank sheet constitution for example.

The idea would be that these are put to the people in a referendum or series of referenda to determine what the people really want - not part of a political package in a general election but standalone options to be investigated, debated and then voted upon.

We can put as much detail or as little in the manifesto as we want... maybe just the promise of a referendum on the future of government in the UK may be enough?

What do people think!?!

**Comment by Floppy** [comment]
Thanks for starting this conversation!

From my point of view, I see a lot of what's in the manifesto as being steps on a journey. There are some immediate fixes to the current system, some mid-term but more major changes, and then the end goal. For instance, based on what's currently in:

Immediate: Change to 3-member STV
Medium: Replace House of Lords with citizen panels, and have a written constitution
Long-term: Move to direct democracy where we don't need representatives in the same way we do now.

Does that make sense? Certainly the more major changes would involve referenda, and a great deal of debate. Perhaps we need to make that more explicit. 

I'm definitely fine with being radical, that's basically the entire point :)

**Comment by andrewdwilliams** [comment]
This is an issue I'm interested in so I'm going to do a bit of research get back to it. I may open another issue depending on how much I find.

**Comment by yellowgopher** [comment]
I am liking the idea of completely seperating the executive from the legislature here - it might even allow for a less party focused legislature...  It has also crossed my mind that we have moved into a period of poor legislation (as referenced in a recent interview with former Cabinet Secretary Gus O'Donnell) and I wonder, with the crossover here (where the executive holds large sway over the legislature) is partly to blame.  I think it is fair to mention that the divorcing of the judiciary from the process of legislation may also have had a part to play - the Supreme Court judges of course no longer sit in the House of Lords and so do not contribute to law making - they have been isolated from government to a certain extent.  I guess, the question is, how do we do it...?

**Comment by Floppy** [comment]
We passed a change suggested a separation of executive and legislature recently: https://github.com/openpolitics/manifesto/pull/313

**Comment by yellowgopher** [comment]
We did!  And it balances out the bias towards the executive as well.
